webapps-backup/src/main/java/io/camunda/webapps/backup/repository/elasticsearch/MetadataMarshaller.java

```diff
@@ -27,7 +26,12 @@ public static Map<String, JsonData> asJson(
   public static Metadata fromMetadata(
       final Map<String, JsonData> metadata, final JsonpMapper jsonpMapper) {
     try {
-      final var backupId = metadata.get("backupId").to(Long.class, jsonpMapper);
+      Long backupId = null;
+      try {
+        backupId = metadata.get("backupId").to(Long.class, jsonpMapper);
```

Comment:
:wrench: If the exception is thrown because `metadata.get("backupId")` returns null, then let's check for that instead of triggering the exception and catching it =/

Reply:
I think the exception can be thrown by the `to` method as well in the tests as the `JsonData` is not null, but the value it wraps is.

Reply:
I added the null check as you suggested, but kept the try/catch for the reason explained in the comment above

Reply:
That's why I asked if it came from `metadata.get` :sweat_smile:

Reply:
I think it may come from both :+1: